lib/dul_hydra/ability_definitions/structure_ability_definitions.rb

```diff
@@ -0,0 +1,13 @@
+module DulHydra
+  class StructureAbilityDefinitions < Ddr::Auth::AbilityDefinitions
+
+    def call
+      can :generate_structure, Ddr::Models::Base do |obj|
```

Comment:
I would consider here instead aliasing `:generate_structure` to `:update`. The other conditions here are not authorization-related.